libbeat/outputs/codecs/json/json.go

```diff
@@ -0,0 +1,54 @@
+package json
```

Comment:
General note: I would prefer if it would be `codec` instead of `codecs` as the package name but that is a general inconsistency we have across beats.